test/utils_shared/mock_utils.py

```diff
@@ -16,7 +16,7 @@
 
 def _build_job_reqs(config, cfgfile, impls):
     with open(cfgfile) as cf:
-        return JobRequirementsResolver(impls[Catalog], cf)
+        return JobRequirementsResolver(cf)
```

Comment:
Now that the JRR no longer depends on the catalog, the notes below about the dependency hack can be removed

Reply:
done

Reply:
:+1: